### PR TITLE
Release2

### PR DESCRIPTION
**Pull request by williamFalcon**
## What does this PR do?

Fixes # (issue)

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/loggers.rst [hunk 1]
```diff
@@ -3,11 +3,139 @@
...
+----------
+
+Custom Logger
+-------------
+
+You can implement your own logger by writing a class that inherits from
+:class:`LightningLoggerBase`. Use the :func:`~pytorch_lightning.loggers.base.rank_zero_only`
+decorator to make sure that only the first process in DDP training logs data.
+
+.. code-block:: python
```
Comment (Borda) [inline docs/source/loggers.rst:45]:
```suggestion
.. testcode::
```

File: docs/source/loggers.rst [hunk 2]
```diff
@@ -3,11 +3,139 @@
...
+
+----------
+
+Using loggers
+-------------
+
+Call the logger anywhere except ``__init__`` in your
+:class:`~pytorch_lightning.core.lightning.LightningModule` by doing:
+
+.. code-block:: python
```
Comment (Borda) [inline docs/source/loggers.rst:84]:
```suggestion
.. testcode::
```

Comment (williamFalcon) [reply docs/source/loggers.rst:84]: no, after release

Comment (Borda) [reply docs/source/loggers.rst:84]: see #2264

**Pull request merged**